app/src/main/res/css/modules/column.sass

```diff
@@ -39,13 +39,15 @@
     margin-top: 5rem
     font-size: 1.3rem
   .items_container
-    overflow-x: hidden
-    overflow-y: scroll
-    overflow-x: hidden
+    position: relative
+    overflow: auto
```

Comment:
@DRSDavidSoft break-all causes some other problems. For example, it also breaks words which look horrible. box-shadow looks fine now so have a look.